<launch>
	<param name="feature_matcher/img_template_name" value="/media/mardava/DATA/Academics/530_707_robot_sys_prog/530_707_independent_project/dat/img_template.jpg" type="string" />
	<param name="feature_matcher/piece_template_name" value="/media/mardava/DATA/Academics/530_707_robot_sys_prog/530_707_independent_project/dat/img_pieces.png" type="string" />
	<node pkg="jps_feature_matching" type="feature_matcher" name="feature_matcher_node" ns="feature_matcher" output="screen" />
</launch>
